flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/WebRuntimeMonitorITCase.java

```diff
@@ -262,8 +247,8 @@ public void testRedirectToLeader() throws Exception {
 				response = followingClient.getNextResponse(deadline.timeLeft());
 				assertEquals(HttpResponseStatus.OK, response.getStatus());
 				assertEquals(response.getType(), MimeTypes.getMimeTypeForExtension("json"));
-				assertTrue(response.getContent().contains("\"taskmanagers\":1") ||
-						response.getContent().contains("\"taskmanagers\":0"));
+				assertThat(response.getContent(), either(containsString("\"taskmanagers\":1"))
```

Comment:
Why did you change this line? (Note that this applies to all lines that replace assertTrue with assertThat, contains with containsString and | with or.)